docs/content/concepts/automation/schedules/testing.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: "Testing schedules | Dagster Docs"
+description: "Learn to test your schedules in Python or the Dagster UI."
+---
+
+# Testing schedules
+
+In this guide, we'll show you how to use the Dagster UI and Python to test your [schedules](/concepts/partitions-schedules-sensors/schedules). For more information about testing in Dagster, refer to the [Testing documentation](/concepts/testing).
+
+---
+
+## Testing schedules in the Dagster UI
+
+Using the UI, you can manually trigger test evaluations of a schedule and view the results. This can be helpful when [creating a schedule](/concepts/automation/schedules/automating-assets-schedules-jobs) or for [troubleshooting unexpected scheduling behavior](/concepts/automation/schedules/troubleshooting).
+
+1. In the UI, click **Overview > Schedules tab**.
+
+2. Click the schedule you want to test.
+
+3. Click the **Test Schedule** button, located near the top right corner of the page.
+
+4. You'll be prompted to select a mock schedule evaluation time. As schedules are defined on a cadence, the evaluation times in the dropdown are past and future times along that cadence.
+
+   For example, let's say you're testing a schedule with a cadence of `"Every day at X time"`. In the dropdown, you'd see past and future evaluation times along that cadence:
+
+   <!-- ![Selecting a mock evaluation time for a schedule in the Dagster UI](/images/concepts/partitions-schedules-sensors/schedules/testing-select-timestamp-page.png) -->
+
+   <Image
+   alt="Selecting a mock evaluation time for a schedule in the Dagster UI"
+   src="/images/concepts/automation/schedules/testing-select-timestamp-page.png"
+   width={2638}
+   height={1388}
+   />
+
+5. After selecting an evaluation time, click the **Evaluate** button.
+
+A window containing the evaluation result will display after the test completes. If the evaluation was successful, click **Open in Launchpad** to launch a run with the same config as the test evaluation.
```

Comment:
Assuming that actually running the job is optional, I think this can highlight that they have a choice to stop after evaluating.

ie.

```suggestion
A window containing the evaluation result will display after the test completes. In this window, you'll see how many runs would potentially be requested if the schedule ran at the selected time. Each of these requests may be configured differently. Here, you can verify that the number of runs and the configuration of the runs match what you would expect.

To drill into the configurations or start a run, click on the buttons in the right column to go to launchpad, where you can investigate each request's configuration and manually run it, if desired.
```